api/users/serializers.py

```diff
@@ -102,6 +102,8 @@ def update(self, instance, validated_data):
                 setattr(instance, attr, value)
         try:
             instance.save()
+        except ValidationError as e:
+            raise InvalidModelValueError(detail=e.args)
```

Comment:
Fixed with kludge. Unfortunately the exception raised has no message attribute. To change this would require updating osf/models/base.save() which I am loath to do.